### PR TITLE
blender-fix: fix for crash in blender 3.3

### PR DESCRIPTION
**Pull request by schinkowski**

### CODE REVIEW

File: Plugins~/Src/MeshSyncClientBlender/msblenBinder.cpp [hunk 1]
```diff
@@ -375,7 +375,12 @@ bool blender::BObject::is_selected() const
 
 Mesh* BObject::to_mesh() const
 {
+#if BLENDER_VERSION >= 303
+    // In blender 3.3 the pointer to interpret the bool moves by 8 instead of 1 for some reason so this needs to be a type of that size:
```
Comment (seandillon92) [inline Plugins~/Src/MeshSyncClientBlender/msblenBinder.cpp:379]: Is this change in Blender 3.3 specific to to_mesh or is it affecting other parts as well?

**Review by sindharta — APPROVED** [review]

**Comment by sindharta** [comment]
Is this fix for crashes that happen when syncing meshes in general (didn't happen to me though) or when syncing geometry nodes only ?

File: Plugins~/Src/MeshSyncClientBlender/msblenBinder.cpp (hunk 1 above)
Comment (schinkowski) [reply Plugins~/Src/MeshSyncClientBlender/msblenBinder.cpp:379]: I only found the issue in this call.

**Comment by sindharta** [comment]
@seandillon92 Can you review this code and give approval if it looks ok ?

**Review by seandillon92 — APPROVED** [review]
Looks good to me

**Pull request merged**